app/assets/stylesheets/modules/landing.scss

```diff
@@ -1,21 +1,60 @@
+@import "variables";
+
+h2 {
+  text-align: center;
+}
+
 .welcome {
+  text-align: center;
```

Comment:
Properties should be ordered background-image, background-repeat, background-size, height, position, text-align